test/ReactWrapper-spec.js

```diff
@@ -139,8 +139,8 @@ describeWithDOM('mount', () => {
       ];
 
       const passes = [
-        <span>Hello</span>,
         <div>Goodbye</div>,
+        <span>More</span>,
```

Comment:
Could you include both tests (with both orderings, i mean) to ensure we don't regress in the future?